jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -1133,6 +1133,7 @@ DeploymentFlow.propTypes = {
   getDiagramURL: PropTypes.func,
   getEntity: PropTypes.func,
   getGithubSSHKeys: PropTypes.func.isRequired,
+  getSLAMachineRates: PropTypes.func.isRequired,
```

Comment:
Wouldn't it be nice to have an OmniAPI shaped up here? Not for this branch of course, just saying...

Reply:
Yes that's a good idea, this component is unwieldy